src/python/fuzzing/strategy.py

```diff
@@ -72,8 +77,15 @@
 LIBFUZZER_STRATEGIES_WITH_PREFIX_VALUE = [
     CORPUS_SUBSET_STRATEGY,
     FORK_STRATEGY,
+    PEACH_GRAMMAR_MUTATION_STRATEGY,
 ]
 
+STRATEGIES_WITH_PREFIX_VALUE_TYPE = {
```

Comment:
nit: append `LIBFUZZER_` to the variable name for consistency

Reply:
Done